external/build/workbox-types.js

```diff
@@ -54,6 +54,12 @@ async function fetchAndPrepare(packages, targetDir) {
 async function run() {
   const t = tmp.dirSync();
   try {
+    // webpack is a peerDependency of workbox-webpack-plugin, and needs to be
+    // manually installed.
+    childProcess.execFileSync('npm', ['install', 'webpack'], {
+      cwd: t.name,
+      stdio: 'inherit',
+    });
```

Comment:
Is there a reason why this can't be specified in `package.json` under the `peerDependencies` key?

Reply:
It is, in the `package.json` for `workbox-webpack-plugin`: https://github.com/GoogleChrome/workbox/blob/b47500cd8a20719f4f2d7f163139408234a70f4a/packages/workbox-webpack-plugin/package.json#L30-L32

`peerDependencies` don't get installed by default when you install an `npm` package—at least not in the version of `npm` that we're using. `npm` versions 1 and 2 did, but versions 3 through 6 (which is currently being used) stopped doing that. Apparently it will start happening again in `npm` version 7. See See https://nodejs.org/es/blog/npm/peer-dependencies/#using-peer-dependencies for more info.